test/system/src/main/java/io/pravega/test/system/framework/services/docker/DockerBasedService.java

```diff
@@ -154,14 +177,15 @@ public boolean isRunning() {
     public CompletableFuture<Void> scaleService(final int instanceCount) {
         try {
             Preconditions.checkArgument(instanceCount >= 0, "negative value: %s", instanceCount);
+            log.info("in scaleservice and instancecount is {}", instanceCount);
```

Comment:
Please, remove all the log messages that you feel that are not needed after merging this PR. In the case you want to add new ones, please make them to start in caps and make them as clear as possible (e.g., `instancecount` -> `instance count`).

Reply:
Fixed